Framework/PythonInterface/mantid/kernel/src/Exports/V3D.cpp

```diff
@@ -63,10 +63,13 @@ void export_V3D() {
       .def(self + self)
       .def(self += self)
       .def(self - self)
+      // cppcheck-suppress duplicateExpression
```

Comment:
What are the errors and can/should they be addressed in this or a separate issue?